src/Akeneo/Platform/Bundle/UIBundle/Resources/workspaces/system/src/hooks/useResetInstance.unit.ts

```diff
@@ -0,0 +1,37 @@
+import {renderHookWithProviders} from '@akeneo-pim-community/legacy-bridge/tests/front/unit/utils';
+import fetchMock from 'jest-fetch-mock';
+import {useResetInstance} from './useResetInstance';
+import {act} from 'react-test-renderer';
+
+const flushPromises = () => new Promise(setImmediate);
+
+test('it launch a reset instance', async () => {
+  fetchMock.mockResponseOnce(async () => Promise.resolve(JSON.stringify([])), {
```

Comment:
You don't need to use `Promise.resolve` if the function is async, javascript will resolve a Promise automatically 
```suggestion
  fetchMock.mockResponseOnce(async () => JSON.stringify([]), {
```